lib/ansible/galaxy/token.py

```diff
@@ -35,6 +48,93 @@
 
 display = Display()
 
+AH_TOKEN_EXPIRES_N_MIN_EARLY = 1
+AH_AUTH_URL = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
+AH_JWKS_URI = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs"
+
+# https://www.rfc-editor.org/rfc/rfc7518#section-3.1
+REQUIRED_JWT_HEADER_KEYS = set(['alg', 'typ', 'kid'])
+REQUIRED_JWT_HEADER_VALUES = {
+    'alg': ['RS256'],
+    'typ': ['JWT'],
+}
+
+PublicNumbers = namedtuple("PublicNumbers", ["e", "n"])
```

Comment:
Is this unused in this module? I only see a single use in the test module. If it's really needed, maybe it should just be in tests?

Reply:
I refactored so I could use test key values more easily and removed this.